File: crabzilla-vavr-example1/crabzilla-vavr-example1-service/src/main/java/crabzilla/example1/services/SampleInternalServiceImpl.java
package crabzilla.example1.services;

import java.time.Instant;
import java.util.UUID;

public class SampleInternalServiceImpl implements SampleInternalService {

  public UUID uuid() {
    return UUID.randomUUID();
  }

  public Instant now() {
    return Instant.now();
  }

}

File: crabzilla-example1/crabzilla-example1-service/src/main/java/io/github/crabzilla/example1/services/SampleInternalServiceImpl.kt
package io.github.crabzilla.example1.services

import io.github.crabzilla.example1.SampleInternalService
import java.time.Instant
import java.util.*

class SampleInternalServiceImpl : SampleInternalService {

  override fun uuid(): UUID {
    return UUID.randomUUID()
  }

  override fun now(): Instant {
    return Instant.now()
  }

}
